assembly/assembly-ide-war/src/main/webapp/_app/IDE.html

```diff
@@ -54,12 +54,278 @@
             }
         };
 
+        /*
+         * Determines whether string starts with specified prefix
+         */
```

Comment:
do we need to define endsWith, startsWith based on browser matrix ?
what are the browsers not supporting it now ?

Reply:
Yes, indeed. Those both functions are already implemented. But when I tried to use them few years ago they was absent.
Tested in Chrome and FF.
Thanks.